src/read.c

```diff
@@ -83,12 +85,7 @@ UInt LoopNesting;
 
 };
 
-static ModuleStateOffset ReaderStateOffset = -1;
-
-extern inline struct ReaderState * ReaderState(void)
-{
-    return (struct ReaderState *)StateSlotsAtOffset(ReaderStateOffset);
-}
+typedef struct ReaderState ReaderState;
```

Comment:
Here we can see that there was a (per-thread) instance of  `struct ReaderState` inside the global GAPState (accessed here via `StateSlotsAtOffset`), and now isn't anymore. `ReaderState` used to refer to a helper function to access it; now `ReaderState` is an alias for `struct ReaderState`, and we instead pass a stack-allocated instance of `struct ReaderState` around via function arguments